src/js/view/view.js

```diff
@@ -903,6 +902,20 @@ function View(_api, _model) {
         }
     };
 
+    this.resizeFloatingPlayer = function() {
+        const html = document.documentElement;
+        const body = document.body;
+        const isMobilePortrait = _isMobile && (html.clientWidth || body.clientWidth) < 420;
```

Comment:
`html.clientWidth || body.clientWidth` is the width of the page. It has nothing to do with whether or not the device is in portrait orientation. It's also expensive to test the page dimensions (this check forces layout).

What are you really trying to do here?

Why `< 420` and not `< MAX_FLOATING_WIDTH`?